### DISCUSSION TITLE
Error 403

### DISCUSSION THREAD

**Discussion by Booteater** [victornpb/undiscord#295] [1. Questions & Answers]
The plugin worked for a while, but eventually stopped working. I'm presented with this error message:

Error searching messages, API responded with status 403!
	{"message":"Missing Access","code":50001}

Any help would be greatly appreciated.

**Comment by Anoneeemous** [comment]
This has happened to me too. I managed to delete around 7k messages in a channel in one of my discords, then it gave the exact same message that the OP has posted.

I am still able to delete messages in different discord servers, but not in the one that I have thus far deleted 7k messages in.

 (has the mee6 bot in there prevented me from deleting any more?)

**Comment by valkyrienyanko** [comment]
You forgot to check "NSFW channel" box.

**Comment by Ishikiopy** [comment]
I have the exact same issue when ever I try to mass delete my messages with the index.json from messages folder of my requested data.

And no this is not caused by NSFW Channel box not being checked or the access token not filled in correctly, both been tested and were correct.
